catroid/src/main/java/org/catrobat/catroid/ui/MainMenuActivity.java

```diff
@@ -82,7 +82,9 @@ protected void onCreate(Bundle savedInstanceState) {
 		boolean hasUserAgreedToPrivacyPolicy = PreferenceManager.getDefaultSharedPreferences(this)
 				.getBoolean(AGREED_TO_PRIVACY_POLICY_PREFERENCE_KEY, false);
 
-		if (hasUserAgreedToPrivacyPolicy) {
+		boolean hasUserUpdatedPocketCode = hasUserUpdatedPocketCode();
```

Comment:
unnecessary boolean field. just call the method inline below, the variable doesnt add any value for the reader since it literally has the same name as the method.